src/ram/analysis/Index.h

```diff
@@ -46,11 +46,14 @@ namespace souffle::ram::analysis {
 
 enum class AttributeConstraint { None, Equal, Inequal };
 
-/** search signature of a RAM operation; each bit represents an attribute of a relation.
- * A one represents that the attribute has an assigned value; a zero represents that
- * no value exists (i.e. attribute is unbounded) in the search. */
+/** Search Signature of a RAM Operation
+ *	Inequal - The attribute has an inequality constraint i.e. 11 <= x <= 13
+ *      Equal   - The attribute has an equality constraint i.e. x = 17
+ *      None    - The attribute no constraint
+ * */
```

Comment:
A tab snuck in, breaking the nice formatting you were trying to add
```suggestion
/** Search Signature of a RAM Operation
 *    Inequal - The attribute has an inequality constraint i.e. 11 <= x <= 13
 *    Equal   - The attribute has an equality constraint i.e. x = 17
 *    None    - The attribute no constraint
 * */
```
(feel free to disagree with my chosen number of spaces for indentation)

Reply:
Nice catch!